README.md

```diff
@@ -1,36 +1,75 @@
 # EnvLogic
+Envlogic is a library used to manage environments for your Ruby application
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/EnvLogic`. To experiment with that code, run `bin/console` for an interactive prompt.
+## Installation
 
-TODO: Delete this and the text above, and describe your gem
+Add gem to your Gemfile
+```ruby
+  gem 'env-logic'
+```
 
-## Installation
+## Usage
 
-Add this line to your application's Gemfile:
+Require env_logic in your app
 
 ```ruby
-gem 'EnvLogic'
+  require 'env_logic'
```

Comment:
what? you should not need this - we will release tjhis as a gem once it is merged and tested so update docs according to this assumption